packages/compass-components/src/components/workspace-tabs/tab.tsx

```diff
@@ -232,8 +237,19 @@ function Tab({
     defaultActionProps
   );
 
+  const { attributes, listeners, setNodeRef, transform, transition } =
+    useSortable({ id: tabIndex });
+  const style = {
+    transform: cssDndKit.Transform.toString(transform),
+    transition,
+  };
+
   return (
     <div
+      ref={setNodeRef}
+      style={style}
+      {...attributes}
+      {...listeners}
```

Comment:
Removed attributes since we do not rely on any explicit value and according to documentation nothing will break if we don't attach them (TS actually pointed at overlapping attributes when I moved them to the same node).